benchmarks/common/benchmarks/toy_detector_benchmark.hpp

```diff
@@ -83,6 +83,11 @@ class ToyDetectorBenchmark : public benchmark::Fixture {
         apply_propagation_config(finding_cfg.propagation);
         apply_propagation_config(fitting_cfg.propagation);
 
+        // Prevent segmentation fault in the Kalman Fitter
+        fitting_cfg.covariance_inflation_factor = 1.f;
```

Comment:
Why is this necessary?

Reply:
I am not sure anymore, because this is from a while back, but I think with this I could recover the tracks that were lost during the fit. I don't want to keep it like that, however, the covariance inflation seems to have a big effect on the the floating point stability. Before it was added, if I remember correctly, the benchmarks did not segfault with the full number of tracks being reconstructed (i.e. setting the grid window to 3 3).